src/controllers/certificates/webhook_reconciler.go

```diff
@@ -67,64 +62,58 @@ func (r *ReconcileWebhookCertificates) Reconcile(ctx context.Context, request re
 	r.namespace = request.Namespace
 	r.ctx = ctx
 
-	secret, err := r.getSecret()
+	mutatingWebhookConfiguration, err := r.getMutatingWebhookConfiguration(ctx)
 	if err != nil {
-		return reconcile.Result{}, err
+		log.Info("could not find mutating webhook configuration, this is normal when deployed using OLM")
```

Comment:
shouldn't this stop the remaining reconciliation from executing?
in the case of OLM the certificates do not need to be created manually

Reply:
the comment on the tests that we are waiting for the certificate secret would make sense here.

should we rethink if always waiting for the certificate secret before startup makes sense?